docs/buildah-bud.md

```diff
@@ -82,6 +177,23 @@ runtime, the manpage to consult is `runc(8)`).
 Note: Do not pass the leading `--` to the flag. To pass the runc flag `--log-format json`
 to buildah bud, the option given would be `--runtime-flag log-format=json`.
 
+**--security-opt**=[]
+
+Security Options
```

Comment:
It would be good to have a sample of this one.

Reply:
fixed